src/SprykerEco/Zed/Computop/Persistence/ComputopEntityManager.php

```diff
@@ -62,4 +66,55 @@ public function updatePaymentComputopOrderItemPaymentConfirmation(
 
         return true;
     }
+
+    /**
+     * @param \Orm\Zed\Computop\Persistence\SpyPaymentComputop $paymentEntity
+     * @param \Spryker\Shared\Kernel\Transfer\TransferInterface $responseTransfer
+     *
+     * @return void
+     */
+    public function savePaymentEntity(
```

Comment:
Change the method in the following way:
`savePaymentResponse(ComputopPayuCeeSingleInitResponse $computopPayuCeeSingleInitResponseTransfer)`.